src/v2/guide/reactivity.md

```diff
@@ -119,10 +119,11 @@ Vue.component('example', {
 })
 ```
 
-Since `$nextTick()` returns a promise, you can achieve the same as the above using the new [ES7 async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) syntax:
+Since `$nextTick()` returns a promise, you can achieve the same as the above using the new [ES2016 async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) syntax:
 
 ```
-...
+Vue.component('example', {
```

Comment:
Hmm, can't you just use ```js for code syntax highlighting?

Reply:
Don't know.. can you?
I am not familiar with the syntax highlighter the vuejs docs use.

Reply:
Try it ;)